src/actions/requests.rs

```diff
@@ -10,7 +10,8 @@
 
 //! Requests that the RLS can respond to.
 
-use actions::{ActionContext, InitActionContext};
+pub use super::execute_command::*;
```

Comment:
I would be happier to just include the exectute_command stuff here, if it is is possible. Otherwise, if we are going to re-export it all, then it would be better to be a submodule of `requests` (and I'd prefer to avoid using a glob re-export).

Reply:
I can just put it in `requests` but it seemed to me that requests is already too big though and `ExecuteCommand` is likely to grow, as it isn't really a single thing. I'd go more toward pulling `CodeAction` out too & putting that together with `ExecuteCommand`. But its not a big deal.